Common/src/config_structure.cpp

```diff
@@ -2445,6 +2445,15 @@ void CConfig::SetConfig_Options() {
   /* DESCRIPTION: Permuting eigenvectors for UQ analysis */
   addBoolOption("UQ_PERMUTE", uq_permute, false);
 
+  /* DESCRIPTION: Number of calls to 'Build' that trigger re-factorization (0 means only once). */
```

Comment:
We need a strategy for the config_template.cpp. Adding these there might be confusing...

Reply:
I prefixed all related options with PASTIX... What I can do is explain those options in an example cfg but not add them to the template config.